src/control/server/storage/bdev_test.go

```diff
@@ -0,0 +1,70 @@
+//
+// (C) Copyright 2022 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package storage
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/daos-stack/daos/src/control/common"
+)
+
+func Test_NvmeDevState(t *testing.T) {
```

Comment:
I wonder if these might make good table-driven tests. The functions checked seem the same for each state.

Reply:
I thought about this but didn't think it was significant value changing, I will address if I have to re-push

Reply:
done